.travis.yml

```diff
@@ -6,6 +6,13 @@ before_install:
   - sudo apt-get install coreutils cppcheck pcregrep python3 python3-pip uncrustify
   - sudo pip3 install flake8
 
+notifications:
+  email:
+    recipients:
+    - maintainers@riot-os.org
```

Comment:
list is actually maintainer@riot-os.org (singular)

Reply:
Fixed